QuizCraft.Domain.API/Services/StatisticsService.cs

```diff
@@ -21,4 +22,26 @@ public async Task<QuizAttemptsDto> QuizAttemptsForUser(string token, Guid quizId
             Attempts = mapper.Map<List<QuizAttemptDto>>(data)
         };
     }
+
+    private static string RetrieveEmail(JwtSecurityTokenHandler jwtSecurityTokenHandler, string token)
+    {
+        var jwtToken = jwtSecurityTokenHandler.ReadJwtToken(token);
+
+        return jwtToken.Claims.First(c => c.Type == "email").Value;
+    }
+
+    public async Task<GlobalStatsDto> GlobalStatisticsAsync()
+    {
+        var totalUsers = await quizRepository.GetTotalUsersAsync();
```

Comment:
In here, is where you would actually calculate the amount of entries, from the records returned, just make sure you receive IEnumerable, meaning that the database command is not executed until the service layer requests the count of records.